test/primeToX.js

```diff
@@ -0,0 +1,26 @@
+const solution = require('../solutions/primeToX.js');
+
+const compareArr=(a1,a2,i=0)=>{
+  if(i==a1.length){
```

Comment:
Be careful of this statement.

If your algorithm prints out [1,2,3] (a1) when the real answer is [1,2,3,4] (a2),
Your algorithm will reach i = 3 and return true.
But the thing is that [1,2,3] and [1,2,3,4] are not equal.